join-us.html

```diff
@@ -189,26 +189,6 @@ <h1 class="display-3 text-center mb-5 mt-4">Frequently Asked Questions</h1>
         </div>
     </section>
 
-    <!-- Sign up section -->
-    <section class="section">
-        <div class="container">
-            <div class="row">
-                <div class="col-md-6">
-                    <img src="assets/img/join_us/img2.svg">
-                </div>
-                <div class="col-md-6 mt-auto mb-auto pl-5">
-                    <div class="display-3 font-weight-light" style="color: #3d317c">Join the <span
-                            class="font-weight-bold d-inline">Team</span></div>
-                    <p class="lead mt-auto mb-auto">I want to join the SEF Team</p>
-                    <a href="apply.html">
```

Comment:
I think we can remove the apply.html file as well

Reply:
Shall we just merge this Pull request and create another issue for removing the `apply.html` file

Reply:
yeah I'll create one @Piumal1999